ccl/changefeedccl: TestAlterChangefeedAddTargetsDuringBackfill failed

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://mesolite.cluster.engflow.com/invocations/default/5bcf2ed8-b4e8-444a-b5bc-3bba90df2b00?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [557a7c4008e689744952ece73adf3fa98a6a25ef](https://github.com/cockroachdb/cockroach/commits/557a7c4008e689744952ece73adf3fa98a6a25ef):


```
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081043322852, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081044180533, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081044180533, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081044180533, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081775447043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081775447043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081775447043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081780054409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081780054409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081780054409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081780577400, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081780577400, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081780577400, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081781361210, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081781361210, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081781361210, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081782005841, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081782005841, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081782005841, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081782725032, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081782725032, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081782725032, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081784040424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081784040424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081784040424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496081785838626, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496081785838626, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496081785838626, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496082871376880, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496082871376880, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496082871376880, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496082873653473, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496082873653473, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496082873653473, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496082891789977, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496082891789977, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739496082891789977, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/165/0}, Timestamp:hlc.Timestamp{WallTime:1739496083907301750, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/165/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739496083907301750, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739496081.775447043,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/kafka
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/165/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/kafka (10.43s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47832

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/e76f9225-be68-40a5-8c67-05dccd5e9817?testReportRun=5&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [22b262749c502d07ec7ccec5b76abd67c361ae4d](https://github.com/cockroachdb/cockroach/commits/22b262749c502d07ec7ccec5b76abd67c361ae4d):


```
    alter_changefeed_test.go:1396: set max span request keys: 104
    alter_changefeed_test.go:1396: set max span request keys: 151
    alter_changefeed_test.go:1396: set max span request keys: 100
    alter_changefeed_test.go:1396: set max span request keys: 192
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/981/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 166
    alter_changefeed_test.go:1396: set max span request keys: 118
    alter_changefeed_test.go:1396: set max span request keys: 130
    alter_changefeed_test.go:1396: set max span request keys: 193
    alter_changefeed_test.go:1396: set max span request keys: 125
    alter_changefeed_test.go:1396: set max span request keys: 104
    alter_changefeed_test.go:1396: set max span request keys: 109
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/100/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/275/0, EndKey:/Table/106/1/440/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/100/0, EndKey:/Table/107/1/266/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/266/0, EndKey:/Table/107/1/384/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/384/0, EndKey:/Table/107/1/514/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/514/0, EndKey:/Table/107/1/707/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/707/0, EndKey:/Table/107/1/832/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/832/0, EndKey:/Table/107/1/936/0}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739520409526153876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739520411172195725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/440/0}, Timestamp:hlc.Timestamp{WallTime:1739520411172195725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/440/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739520411172195725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739520411172195725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739520413782847291, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/440/0}, Timestamp:hlc.Timestamp{WallTime:1739520413782847291, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/440/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739520413782847291, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739520413782847291, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739520413949807914, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/440/0}, Timestamp:hlc.Timestamp{WallTime:1739520413949807914, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/440/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739520413949807914, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739520413949807914, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739520413958649529, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/440/0}, Timestamp:hlc.Timestamp{WallTime:1739520413958649529, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/440/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739520413958649529, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739520413983048219, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739520409.526153876,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pubsub
        	Messages:   	span should not have been resolved: /Table/106/1{-/101/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pubsub (9.51s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=5</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18834476?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18834476?buildTab=artifacts#/) on master @ [5529b716ba22537177bcae2c1fc8319ed083631b](https://github.com/cockroachdb/cockroach/commits/5529b716ba22537177bcae2c1fc8319ed083631b):


```
    alter_changefeed_test.go:1396: set max span request keys: 129
    alter_changefeed_test.go:1396: set max span request keys: 178
    alter_changefeed_test.go:1396: set max span request keys: 147
    alter_changefeed_test.go:1396: set max span request keys: 158
    alter_changefeed_test.go:1396: set max span request keys: 187
    alter_changefeed_test.go:1396: set max span request keys: 123
    alter_changefeed_test.go:1396: set max span request keys: 106
    helpers_test.go:734: waiting for checkpoint
    helpers_test.go:737: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/NULL}, roachpb.Span{Key:/Tenant/10/Table/106/1/129/0, EndKey:/Tenant/10/Table/106/1/922/0}}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 154
    alter_changefeed_test.go:1396: set max span request keys: 168
    alter_changefeed_test.go:1396: set max span request keys: 129
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/129/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 115
    alter_changefeed_test.go:1396: set max span request keys: 174
    alter_changefeed_test.go:1396: set max span request keys: 162
    alter_changefeed_test.go:1396: set max span request keys: 189
    alter_changefeed_test.go:1396: set max span request keys: 165
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 196
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/922/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/154/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/154/0, EndKey:/Tenant/10/Table/107/1/269/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/269/0, EndKey:/Tenant/10/Table/107/1/443/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/443/0, EndKey:/Tenant/10/Table/107/1/605/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/605/0, EndKey:/Tenant/10/Table/107/1/794/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/794/0, EndKey:/Tenant/10/Table/107/1/959/0}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739528406597090222, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/129/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/129/0}, Timestamp:hlc.Timestamp{WallTime:1739528413511072725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/129/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739528413511072725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739528413511072725, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/129/0}, Timestamp:hlc.Timestamp{WallTime:1739528414786247192, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/129/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739528414786247192, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739528414786247192, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/129/0}, Timestamp:hlc.Timestamp{WallTime:1739528417875603004, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/129/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739528417875603004, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739528417875603004, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739528414.786247192,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pubsub
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/129/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pubsub (16.74s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18834919?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18834919?buildTab=artifacts#/) on master @ [5529b716ba22537177bcae2c1fc8319ed083631b](https://github.com/cockroachdb/cockroach/commits/5529b716ba22537177bcae2c1fc8319ed083631b):


```
    helpers_test.go:737: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/NULL}, roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/1/844/0}}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 177
    alter_changefeed_test.go:1396: set max span request keys: 113
    alter_changefeed_test.go:1396: set max span request keys: 150
    alter_changefeed_test.go:1396: set max span request keys: 175
    alter_changefeed_test.go:1396: set max span request keys: 149
    alter_changefeed_test.go:1396: set max span request keys: 125
    alter_changefeed_test.go:1396: set max span request keys: 185
    alter_changefeed_test.go:1396: set max span request keys: 196
    alter_changefeed_test.go:1396: set max span request keys: 159
    alter_changefeed_test.go:1396: set max span request keys: 106
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/150/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 188
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/844/0, EndKey:/Table/106/1/957/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/844/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/177/0, EndKey:/Table/107/1/302/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/302/0, EndKey:/Table/107/1/487/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/487/0, EndKey:/Table/107/1/683/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/683/0, EndKey:/Table/107/1/842/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/842/0, EndKey:/Table/107/1/948/0}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739531790778579113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531792682710792, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531792682710792, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739531792682710792, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531795088750333, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531795088750333, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739531795088750333, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531795229508656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531795229508656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739531795229508656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531795229723298, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531795229723298, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739531795229723298, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/155/0}, Timestamp:hlc.Timestamp{WallTime:1739531795237658782, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739531790.778579113,0
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/155/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739531795237658782, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/kafka
        	Messages:   	span should not have been resolved: /Table/106/1{-/155/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/kafka (6.21s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/e83f8778-8f1d-4dc7-ac61-8ac347a33058?testReportRun=3&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [5529b716ba22537177bcae2c1fc8319ed083631b](https://github.com/cockroachdb/cockroach/commits/5529b716ba22537177bcae2c1fc8319ed083631b):


```
    alter_changefeed_test.go:1396: set max span request keys: 132
    alter_changefeed_test.go:1396: set max span request keys: 135
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/116/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 119
    alter_changefeed_test.go:1396: set max span request keys: 130
    alter_changefeed_test.go:1396: set max span request keys: 124
    alter_changefeed_test.go:1396: set max span request keys: 167
    alter_changefeed_test.go:1396: set max span request keys: 169
    alter_changefeed_test.go:1396: set max span request keys: 186
    alter_changefeed_test.go:1396: set max span request keys: 111
    alter_changefeed_test.go:1396: set max span request keys: 146
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/876/0, EndKey:/Tenant/10/Table/106/1/987/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/355/0, EndKey:/Tenant/10/Table/106/1/485/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/876/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/355/0, EndKey:/Tenant/10/Table/106/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/130/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/130/0, EndKey:/Tenant/10/Table/107/1/254/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/254/0, EndKey:/Tenant/10/Table/107/1/421/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/421/0, EndKey:/Tenant/10/Table/107/1/590/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/590/0, EndKey:/Tenant/10/Table/107/1/776/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/776/0, EndKey:/Tenant/10/Table/107/1/887/0}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739538018029144332, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739538021625626324, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/187/0, EndKey:/Tenant/10/Table/106/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739538021625626324, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/591/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739538021625626324, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739538021625626324, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739538022647983241, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739538022647983241, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/187/0, EndKey:/Tenant/10/Table/106/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739538022647983241, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/591/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739538022647983241, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739538025553062648, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739538025553062648, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/187/0, EndKey:/Tenant/10/Table/106/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739538025553062648, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/591/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739538025553062648, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739538028614649594, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/187/0, EndKey:/Tenant/10/Table/106/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739538028614649594, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739538028614649594, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/591/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739538028614649594, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739538021.625626324,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/187/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (30.29s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18835887?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18835887?buildTab=artifacts#/) on master @ [1e0e0df04867b587b88ad041a5ea127ce08cd5f0](https://github.com/cockroachdb/cockroach/commits/1e0e0df04867b587b88ad041a5ea127ce08cd5f0):


```
    alter_changefeed_test.go:1396: set max span request keys: 137
    alter_changefeed_test.go:1396: set max span request keys: 145
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/789/0, EndKey:/Table/106/1/938/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 183
    alter_changefeed_test.go:1396: set max span request keys: 181
    alter_changefeed_test.go:1396: set max span request keys: 135
    alter_changefeed_test.go:1396: set max span request keys: 159
    alter_changefeed_test.go:1396: set max span request keys: 108
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/290/0, EndKey:/Table/106/1/406/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/290/0, EndKey:/Table/106/1/414/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/117/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/789/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/137/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/137/0, EndKey:/Table/107/1/282/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/282/0, EndKey:/Table/107/1/465/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/465/0, EndKey:/Table/107/1/646/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/646/0, EndKey:/Table/107/1/781/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/781/0, EndKey:/Table/107/1/940/0}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546318387885508, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739546320058924223, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/177/0, EndKey:/Table/106/1/414/0}, Timestamp:hlc.Timestamp{WallTime:1739546320058924223, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/414/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739546320058924223, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546320058924223, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739546322666751064, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/177/0, EndKey:/Table/106/1/414/0}, Timestamp:hlc.Timestamp{WallTime:1739546322666751064, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/414/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739546322666751064, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546322666751064, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739546322699042876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/177/0, EndKey:/Table/106/1/414/0}, Timestamp:hlc.Timestamp{WallTime:1739546322699042876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/414/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739546322699042876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546322699042876, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546322703084609, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/177/0}, Timestamp:hlc.Timestamp{WallTime:1739546322708706755, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/177/0, EndKey:/Table/106/1/414/0}, Timestamp:hlc.Timestamp{WallTime:1739546322708706755, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/414/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739546322708706755, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739546322708706755, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739546318.387885508,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pulsar
        	Messages:   	span should not have been resolved: /Table/106/1{-/177/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pulsar (6.74s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18836780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18836780?buildTab=artifacts#/) on master @ [a4c43db275b5d74fbee930086ede6d2c22a7346d](https://github.com/cockroachdb/cockroach/commits/a4c43db275b5d74fbee930086ede6d2c22a7346d):


```
    helpers_test.go:737: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/NULL}, roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/1/945/0}}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 123
    alter_changefeed_test.go:1396: set max span request keys: 105
    alter_changefeed_test.go:1396: set max span request keys: 198
    alter_changefeed_test.go:1396: set max span request keys: 175
    alter_changefeed_test.go:1396: set max span request keys: 140
    alter_changefeed_test.go:1396: set max span request keys: 147
    alter_changefeed_test.go:1396: set max span request keys: 195
    alter_changefeed_test.go:1396: set max span request keys: 191
    alter_changefeed_test.go:1396: set max span request keys: 121
    alter_changefeed_test.go:1396: set max span request keys: 153
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/123/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/123/0, EndKey:/Table/107/1/228/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/228/0, EndKey:/Table/107/1/426/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/426/0, EndKey:/Table/107/1/601/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/601/0, EndKey:/Table/107/1/741/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/741/0, EndKey:/Table/107/1/888/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/945/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/121/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549753123912774, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549754944581730, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549754944581730, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549754944581730, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549757354988819, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549757354988819, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549757354988819, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549757495738663, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549757495738663, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549757495738663, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549757495926884, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549757495926884, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549757495926884, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/125/0}, Timestamp:hlc.Timestamp{WallTime:1739549757504273445, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/125/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739549757504273445, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739549757557893643, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739549753.123912774,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/kafka
        	Messages:   	span should not have been resolved: /Table/106/1{-/125/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/kafka (9.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18837214?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18837214?buildTab=artifacts#/) on master @ [9f7bab198c51af6d4c9c5dba25e8a52f04ed6b82](https://github.com/cockroachdb/cockroach/commits/9f7bab198c51af6d4c9c5dba25e8a52f04ed6b82):


```
    helpers_test.go:749: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 109
    alter_changefeed_test.go:1396: set max span request keys: 194
    alter_changefeed_test.go:1396: set max span request keys: 121
    alter_changefeed_test.go:1396: set max span request keys: 184
    alter_changefeed_test.go:1396: set max span request keys: 144
    alter_changefeed_test.go:1396: set max span request keys: 149
    alter_changefeed_test.go:1396: set max span request keys: 137
    alter_changefeed_test.go:1396: set max span request keys: 175
    alter_changefeed_test.go:1396: set max span request keys: 167
    alter_changefeed_test.go:1396: set max span request keys: 188
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/109/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/998/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/121/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/109/0, EndKey:/Table/107/1/303/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/303/0, EndKey:/Table/107/1/452/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/452/0, EndKey:/Table/107/1/589/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/589/0, EndKey:/Table/107/1/764/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/764/0, EndKey:/Table/107/1/931/0}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551783542695277, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551785939458278, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551785939458278, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551785939458278, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551787748333424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551787748333424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551787748333424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551787763517018, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551787763517018, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551787763517018, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551787763678672, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551787763678672, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551787763678672, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551787774435321, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739551787774435321, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551787774435321, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/168/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739551788781437505, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/168/0}, Timestamp:hlc.Timestamp{WallTime:1739551788781437505, Logical:0}, BoundaryType:0}
    helpers_test.go:749: waiting for highwater
    helpers_test.go:752: read highwater: 1739551783.542695277,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1202
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Table/106/1{-/168/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (7.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18842089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18842089?buildTab=artifacts#/) on master @ [d799af1e502a03956f175b50161961ffc1d9495a](https://github.com/cockroachdb/cockroach/commits/d799af1e502a03956f175b50161961ffc1d9495a):


```
    helpers_test.go:735: waiting for checkpoint
    helpers_test.go:735: waiting for checkpoint
    alter_changefeed_test.go:1396: set max span request keys: 104
    alter_changefeed_test.go:1396: set max span request keys: 142
    alter_changefeed_test.go:1396: set max span request keys: 180
    alter_changefeed_test.go:1396: set max span request keys: 143
    alter_changefeed_test.go:1396: set max span request keys: 196
    alter_changefeed_test.go:1396: set max span request keys: 152
    alter_changefeed_test.go:1396: set max span request keys: 160
    helpers_test.go:735: waiting for checkpoint
    helpers_test.go:738: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/NULL}, roachpb.Span{Key:/Tenant/10/Table/106/1/246/0, EndKey:/Tenant/10/Table/106/1/917/0}}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 191
    alter_changefeed_test.go:1396: set max span request keys: 111
    alter_changefeed_test.go:1396: set max span request keys: 181
    alter_changefeed_test.go:1396: set max span request keys: 173
    alter_changefeed_test.go:1396: set max span request keys: 182
    alter_changefeed_test.go:1396: set max span request keys: 177
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/111/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 199
    alter_changefeed_test.go:1396: set max span request keys: 143
    alter_changefeed_test.go:1396: set max span request keys: 124
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/917/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/191/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/191/0, EndKey:/Tenant/10/Table/107/1/373/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/246/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/373/0, EndKey:/Tenant/10/Table/107/1/550/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/550/0, EndKey:/Tenant/10/Table/107/1/749/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/749/0, EndKey:/Tenant/10/Table/107/1/892/0}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739602912095853656, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/246/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/246/0}, Timestamp:hlc.Timestamp{WallTime:1739602915081848261, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/246/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739602915081848261, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739602915081848261, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/246/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739602915353305731, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/246/0}, Timestamp:hlc.Timestamp{WallTime:1739602915353305731, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/246/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739602917296967983, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739602917296967983, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/246/0}, Timestamp:hlc.Timestamp{WallTime:1739602917296967983, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739602912.095853656,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/246/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (11.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/47789650-314c-4788-b646-0774ce7dc568?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [67a82bfc09766413d7f8ae8edb10035940b94c7e](https://github.com/cockroachdb/cockroach/commits/67a82bfc09766413d7f8ae8edb10035940b94c7e):


```
    alter_changefeed_test.go:1396: set max span request keys: 128
    alter_changefeed_test.go:1396: set max span request keys: 199
    alter_changefeed_test.go:1396: set max span request keys: 106
    alter_changefeed_test.go:1396: set max span request keys: 132
    alter_changefeed_test.go:1396: set max span request keys: 193
    helpers_test.go:735: waiting for checkpoint
    helpers_test.go:738: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/NULL}, roachpb.Span{Key:/Tenant/10/Table/106/1/103/0, EndKey:/Tenant/10/Table/106/1/262/0}, roachpb.Span{Key:/Tenant/10/Table/106/1/390/0, EndKey:/Tenant/10/Table/106/1/827/0}}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 109
    alter_changefeed_test.go:1396: set max span request keys: 155
    alter_changefeed_test.go:1396: set max span request keys: 173
    alter_changefeed_test.go:1396: set max span request keys: 122
    alter_changefeed_test.go:1396: set max span request keys: 112
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/103/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 125
    alter_changefeed_test.go:1396: set max span request keys: 187
    alter_changefeed_test.go:1396: set max span request keys: 108
    alter_changefeed_test.go:1396: set max span request keys: 186
    alter_changefeed_test.go:1396: set max span request keys: 121
    alter_changefeed_test.go:1396: set max span request keys: 165
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/827/0, EndKey:/Tenant/10/Table/106/1/936/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/262/0, EndKey:/Tenant/10/Table/106/1/390/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/122/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/827/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/122/0, EndKey:/Tenant/10/Table/107/1/247/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/247/0, EndKey:/Tenant/10/Table/107/1/434/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/434/0, EndKey:/Tenant/10/Table/107/1/542/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/542/0, EndKey:/Tenant/10/Table/107/1/728/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/728/0, EndKey:/Tenant/10/Table/107/1/849/0}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739605161526215671, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/103/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/103/0}, Timestamp:hlc.Timestamp{WallTime:1739605169316785777, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/103/0, EndKey:/Tenant/10/Table/106/1/390/0}, Timestamp:hlc.Timestamp{WallTime:1739605169316785777, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/390/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739605169316785777, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739605169316785777, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/390/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739605171125439050, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/103/0}, Timestamp:hlc.Timestamp{WallTime:1739605171125439050, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/103/0, EndKey:/Tenant/10/Table/106/1/390/0}, Timestamp:hlc.Timestamp{WallTime:1739605171125439050, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739605171125439050, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739605161.526215671,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pubsub
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/103/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pubsub (13.34s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/7419118f-1631-4484-93dc-46c8665d8352?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [d799af1e502a03956f175b50161961ffc1d9495a](https://github.com/cockroachdb/cockroach/commits/d799af1e502a03956f175b50161961ffc1d9495a):


```
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/175/0}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/175/0, EndKey:/Table/107/1/354/0}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/354/0, EndKey:/Table/107/1/519/0}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/519/0, EndKey:/Table/107/1/660/0}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/660/0, EndKey:/Table/107/1/818/0}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611294113557603, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611296539787028, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611296539787028, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611296539787028, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611296539787028, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611297566387243, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611297566387243, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611297566387243, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611297566387243, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611297747667113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611297747667113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611297747667113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611297747667113, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611297751131632, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611297751131632, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611297751131632, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611297751131632, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611297795758719, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611297795758719, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611297795758719, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611297795758719, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611298814306897, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611298814306897, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611298814306897, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611300618056409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611300618056409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611300618056409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611300618056409, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/101/0}, Timestamp:hlc.Timestamp{WallTime:1739611303656050904, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/101/0, EndKey:/Table/106/1/692/0}, Timestamp:hlc.Timestamp{WallTime:1739611303656050904, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/692/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739611303656050904, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739611303656050904, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739611300.618056409,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pubsub
        	Messages:   	span should not have been resolved: /Table/106/1{-/101/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pubsub (18.72s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18844114?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18844114?buildTab=artifacts#/) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    alter_changefeed_test.go:1396: set max span request keys: 185
    alter_changefeed_test.go:1396: set max span request keys: 190
    alter_changefeed_test.go:1396: set max span request keys: 180
    alter_changefeed_test.go:1396: set max span request keys: 185
    alter_changefeed_test.go:1396: set max span request keys: 153
    alter_changefeed_test.go:1396: set max span request keys: 158
    alter_changefeed_test.go:1396: set max span request keys: 131
    alter_changefeed_test.go:1396: set max span request keys: 116
    alter_changefeed_test.go:1396: set max span request keys: 151
    alter_changefeed_test.go:1396: set max span request keys: 180
    alter_changefeed_test.go:1396: set max span request keys: 146
    alter_changefeed_test.go:1396: set max span request keys: 162
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/185/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/185/0, EndKey:/Table/107/1/365/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/506/0, EndKey:/Table/106/1/691/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/691/0, EndKey:/Table/106/1/849/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/849/0, EndKey:/Table/106/1/980/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/365/0, EndKey:/Table/107/1/518/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/506/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/518/0, EndKey:/Table/107/1/669/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/669/0, EndKey:/Table/107/1/849/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/849/0, EndKey:/Table/107/1/995/0}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739652883444797091, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652885618492809, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/135/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652885618492809, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739652885618492809, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652887440622557, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/135/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652887440622557, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739652887440622557, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652887619195555, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/135/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652887619195555, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739652887619195555, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652887643217033, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/135/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652887643217033, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739652887643217033, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/135/0}, Timestamp:hlc.Timestamp{WallTime:1739652888644656636, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/135/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739652888644656636, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739652883.444797091,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pubsub
        	Messages:   	span should not have been resolved: /Table/106/1{-/135/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pubsub (7.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18844404?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18844404?buildTab=artifacts#/) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    helpers_test.go:735: waiting for checkpoint
    helpers_test.go:738: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/NULL}, roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/1/852/0}}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 123
    alter_changefeed_test.go:1396: set max span request keys: 159
    alter_changefeed_test.go:1396: set max span request keys: 198
    alter_changefeed_test.go:1396: set max span request keys: 153
    alter_changefeed_test.go:1396: set max span request keys: 184
    alter_changefeed_test.go:1396: set max span request keys: 121
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/123/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 121
    alter_changefeed_test.go:1396: set max span request keys: 166
    alter_changefeed_test.go:1396: set max span request keys: 127
    alter_changefeed_test.go:1396: set max span request keys: 149
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/123/0, EndKey:/Table/107/1/276/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/276/0, EndKey:/Table/107/1/460/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/852/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/460/0, EndKey:/Table/107/1/581/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/581/0, EndKey:/Table/107/1/702/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/702/0, EndKey:/Table/107/1/868/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/868/0, EndKey:/Table/107/1/995/0}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739655229177587096, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655231310124851, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655231310124851, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739655231310124851, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655233514996432, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655233514996432, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739655233514996432, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655233627766675, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655233627766675, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739655233627766675, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655233635910165, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655233635910165, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739655233635910165, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/148/0}, Timestamp:hlc.Timestamp{WallTime:1739655234643085159, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/148/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739655234643085159, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739655233.627766675,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Table/106/1{-/148/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (7.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/c8b30906-1529-446f-bc0e-f9f80109b37f?testReportRun=3&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 199
    alter_changefeed_test.go:1396: set max span request keys: 162
    alter_changefeed_test.go:1396: set max span request keys: 136
    alter_changefeed_test.go:1396: set max span request keys: 139
    alter_changefeed_test.go:1396: set max span request keys: 113
    alter_changefeed_test.go:1396: set max span request keys: 197
    alter_changefeed_test.go:1396: set max span request keys: 195
    alter_changefeed_test.go:1396: set max span request keys: 190
    alter_changefeed_test.go:1396: set max span request keys: 107
    alter_changefeed_test.go:1396: set max span request keys: 162
    alter_changefeed_test.go:1396: set max span request keys: 117
    alter_changefeed_test.go:1396: set max span request keys: 115
    alter_changefeed_test.go:1396: set max span request keys: 142
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/669/0, EndKey:/Tenant/10/Table/106/1/831/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/199/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/831/0, EndKey:/Tenant/10/Table/106/1/970/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/249/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/669/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/199/0, EndKey:/Tenant/10/Table/107/1/396/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/396/0, EndKey:/Tenant/10/Table/107/1/503/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/503/0, EndKey:/Tenant/10/Table/107/1/665/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/665/0, EndKey:/Tenant/10/Table/107/1/782/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/782/0, EndKey:/Tenant/10/Table/107/1/897/0}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739698176297384453, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:1739698182769643899, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/348/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739698182769643899, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739698182769643899, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:1739698183790112445, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/348/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739698183790112445, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739698183790112445, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:1739698186639689593, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/348/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739698186639689593, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739698186639689593, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/348/0}, Timestamp:hlc.Timestamp{WallTime:1739698189684699454, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/348/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739698189684699454, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739698189684699454, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739698186.639689593,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/348/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (22.96s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/5392e913-fafd-474c-8865-35508a4e2e81?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    alter_changefeed_test.go:1396: set max span request keys: 142
    helpers_test.go:735: waiting for checkpoint
    helpers_test.go:738: read checkpoint: &jobspb.ChangefeedProgress_Checkpoint{Spans:[]roachpb.Span{roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/NULL}, roachpb.Span{Key:/Table/106/1/324/0, EndKey:/Table/106/1/951/0}}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 190
    alter_changefeed_test.go:1396: set max span request keys: 159
    alter_changefeed_test.go:1396: set max span request keys: 182
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/951/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 131
    alter_changefeed_test.go:1396: set max span request keys: 104
    alter_changefeed_test.go:1396: set max span request keys: 135
    alter_changefeed_test.go:1396: set max span request keys: 105
    alter_changefeed_test.go:1396: set max span request keys: 153
    alter_changefeed_test.go:1396: set max span request keys: 186
    alter_changefeed_test.go:1396: set max span request keys: 190
    alter_changefeed_test.go:1396: set max span request keys: 180
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/190/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/182/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/182/0, EndKey:/Table/107/1/286/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/286/0, EndKey:/Table/107/1/421/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/190/0, EndKey:/Table/106/1/321/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/421/0, EndKey:/Table/107/1/526/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/526/0, EndKey:/Table/107/1/712/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/712/0, EndKey:/Table/107/1/902/0}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739711497142164051, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:1739711500448318930, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/324/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739711500448318930, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739711500448318930, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:1739711502740505451, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/324/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739711502740505451, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739711502740505451, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:1739711505773973424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/324/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739711505773973424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739711505773973424, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/324/0}, Timestamp:hlc.Timestamp{WallTime:1739711508639520056, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/324/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739711508639520056, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739711508639520056, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739711505.773973424,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/pulsar
        	Messages:   	span should not have been resolved: /Table/106/1{-/324/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/pulsar (20.40s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/586a3bfa-d23b-4ab4-b6ff-54b7226d371d?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1396: set max span request keys: 137
    alter_changefeed_test.go:1396: set max span request keys: 139
    alter_changefeed_test.go:1396: set max span request keys: 136
    alter_changefeed_test.go:1396: set max span request keys: 117
    alter_changefeed_test.go:1396: set max span request keys: 193
    alter_changefeed_test.go:1396: set max span request keys: 137
    alter_changefeed_test.go:1396: set max span request keys: 111
    alter_changefeed_test.go:1396: set max span request keys: 162
    alter_changefeed_test.go:1396: set max span request keys: 126
    alter_changefeed_test.go:1396: set max span request keys: 129
    alter_changefeed_test.go:1396: set max span request keys: 100
    alter_changefeed_test.go:1396: set max span request keys: 128
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1396: set max span request keys: 172
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/137/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/820/0, EndKey:/Tenant/10/Table/106/1/959/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/820/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/253/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/137/0, EndKey:/Tenant/10/Table/107/1/330/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/330/0, EndKey:/Tenant/10/Table/107/1/492/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/492/0, EndKey:/Tenant/10/Table/107/1/618/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/618/0, EndKey:/Tenant/10/Table/107/1/747/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/747/0, EndKey:/Tenant/10/Table/107/1/847/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/847/0, EndKey:/Tenant/10/Table/107/1/975/0}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739783252521446629, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:1739783258828556043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/317/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739783258828556043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739783258828556043, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:1739783259876353821, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/317/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739783259876353821, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739783259876353821, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:1739783262930928628, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/317/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739783262930928628, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739783262930928628, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/317/0}, Timestamp:hlc.Timestamp{WallTime:1739783265769210201, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/317/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739783265769210201, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739783265769210201, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739783262.930928628,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/{1/317/0-2}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (27.07s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18848435?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18848435?buildTab=artifacts#/) on master @ [867043b4f58381f84d1da0c55de92b7183856c70](https://github.com/cockroachdb/cockroach/commits/867043b4f58381f84d1da0c55de92b7183856c70):


```
    alter_changefeed_test.go:1396: set max span request keys: 114
    alter_changefeed_test.go:1396: set max span request keys: 179
    alter_changefeed_test.go:1396: set max span request keys: 196
    alter_changefeed_test.go:1396: set max span request keys: 199
    alter_changefeed_test.go:1396: set max span request keys: 123
    alter_changefeed_test.go:1396: set max span request keys: 198
    alter_changefeed_test.go:1396: set max span request keys: 136
    alter_changefeed_test.go:1396: set max span request keys: 158
    alter_changefeed_test.go:1396: set max span request keys: 184
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/690/0, EndKey:/Table/106/1/800/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/385/0, EndKey:/Table/106/1/502/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/800/0, EndKey:/Table/106/1/988/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/114/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/385/0, EndKey:/Table/106/1/509/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/690/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/114/0, EndKey:/Table/107/1/313/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/313/0, EndKey:/Table/107/1/436/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/436/0, EndKey:/Table/107/1/634/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/634/0, EndKey:/Table/107/1/770/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/770/0, EndKey:/Table/107/1/928/0}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739791992926407645, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:1739791994666619993, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/104/0, EndKey:/Table/106/1/509/0}, Timestamp:hlc.Timestamp{WallTime:1739791994666619993, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/509/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739791994666619993, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739791994666619993, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739791997279390407, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:1739791997279390407, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/104/0, EndKey:/Table/106/1/509/0}, Timestamp:hlc.Timestamp{WallTime:1739791997279390407, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/509/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739791997279390407, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739791997286137430, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:1739791997286137430, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/104/0, EndKey:/Table/106/1/509/0}, Timestamp:hlc.Timestamp{WallTime:1739791997286137430, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/509/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739791997286137430, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/104/0}, Timestamp:hlc.Timestamp{WallTime:1739791997299621370, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/104/0, EndKey:/Table/106/1/509/0}, Timestamp:hlc.Timestamp{WallTime:1739791997299621370, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/509/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739791997299621370, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739791997299621370, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739791992.926407645,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/kafka
        	Messages:   	span should not have been resolved: /Table/106/1{-/104/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/kafka (9.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://tanzanite.cluster.engflow.com/invocations/default/752fb15b-34aa-48f6-80a4-4e56d09d9dd2?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [34e34c2d5b9598441b935f03a794277e9329a256](https://github.com/cockroachdb/cockroach/commits/34e34c2d5b9598441b935f03a794277e9329a256):


```
    alter_changefeed_test.go:1396: set max span request keys: 114
    alter_changefeed_test.go:1396: set max span request keys: 115
    alter_changefeed_test.go:1396: set max span request keys: 197
    alter_changefeed_test.go:1396: set max span request keys: 159
    alter_changefeed_test.go:1396: set max span request keys: 147
    alter_changefeed_test.go:1396: set max span request keys: 173
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/612/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/1/141/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/NULL, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/141/0, EndKey:/Table/107/1/255/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/255/0, EndKey:/Table/107/1/370/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/370/0, EndKey:/Table/107/1/567/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/567/0, EndKey:/Table/107/1/726/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1/726/0, EndKey:/Table/107/1/873/0}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789847129506111, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789849400504787, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789849400504787, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/774/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739789849400504787, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789849400504787, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/774/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739789851235254307, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789851235254307, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789851235254307, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789851235254307, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789851394644700, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789851394644700, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/774/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739789851394644700, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789851394644700, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789851409298618, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789851409298618, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789851445923852, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789851445923852, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/774/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739789851445923852, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789851445923852, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/172/0, EndKey:/Table/106/1/774/0}, Timestamp:hlc.Timestamp{WallTime:1739789851536785936, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1/774/0, EndKey:/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739789851536785936, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/106/1, EndKey:/Table/106/1/172/0}, Timestamp:hlc.Timestamp{WallTime:1739789851536785936, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Table/107/1, EndKey:/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739789851536785936, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    helpers_test.go:753: read highwater: 1739789847.129506111,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage
        	Messages:   	span should not have been resolved: /Table/106/1{-/172/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/cloudstorage (13.99s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringBackfill [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18849481?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18849481?buildTab=artifacts#/) on master @ [b64e11135c5deaea3ce874b46c518993e4619390](https://github.com/cockroachdb/cockroach/commits/b64e11135c5deaea3ce874b46c518993e4619390):


```
    alter_changefeed_test.go:1396: set max span request keys: 136
    alter_changefeed_test.go:1396: set max span request keys: 105
    alter_changefeed_test.go:1396: set max span request keys: 155
    alter_changefeed_test.go:1396: set max span request keys: 146
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/977/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/551/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/NULL, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/1/187/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/187/0, EndKey:/Tenant/10/Table/107/1/322/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/322/0, EndKey:/Tenant/10/Table/107/1/426/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/426/0, EndKey:/Tenant/10/Table/107/1/591/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/591/0, EndKey:/Tenant/10/Table/107/1/727/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/727/0, EndKey:/Tenant/10/Table/107/1/832/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1/832/0, EndKey:/Tenant/10/Table/107/1/987/0}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804985330595119, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804987767202848, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804987767202848, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804987767202848, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804987767202848, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804988122159045, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804988122159045, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804988122159045, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804988122159045, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804988123414304, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804988123414304, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804988123414304, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804988123414304, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804988124336197, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804988124336197, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804988124336197, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:0, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804988167995169, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804989147384151, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804989147384151, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804989147384151, Logical:0}, BoundaryType:0}
    helpers_test.go:750: waiting for highwater
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1, EndKey:/Tenant/10/Table/106/1/136/0}, Timestamp:hlc.Timestamp{WallTime:1739804991040443292, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/136/0, EndKey:/Tenant/10/Table/106/1/655/0}, Timestamp:hlc.Timestamp{WallTime:1739804991040443292, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/106/1/655/0, EndKey:/Tenant/10/Table/106/2}, Timestamp:hlc.Timestamp{WallTime:1739804991040443292, Logical:0}, BoundaryType:0}
    alter_changefeed_test.go:1472: resolved span: &jobspb.ResolvedSpan{Span:roachpb.Span{Key:/Tenant/10/Table/107/1, EndKey:/Tenant/10/Table/107/2}, Timestamp:hlc.Timestamp{WallTime:1739804991040443292, Logical:0}, BoundaryType:0}
    helpers_test.go:753: read highwater: 1739804987.767202848,0
    alter_changefeed_test.go:1493: 
        	Error Trace:	pkg/ccl/changefeedccl/alter_changefeed_test.go:1493
        	            				pkg/ccl/changefeedccl/helpers_test.go:1203
        	Error:      	Should be false
        	Test:       	TestAlterChangefeedAddTargetsDuringBackfill/kafka
        	Messages:   	span should not have been resolved: /Tenant/10/Table/106/1{-/136/0}
    --- FAIL: TestAlterChangefeedAddTargetsDuringBackfill/kafka (10.86s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringBackfill.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/alter_changefeed_test.go
+++ b/pkg/ccl/changefeedccl/alter_changefeed_test.go
@@ -32,13 +32,13 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/execinfra"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/ctxgroup"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/randutil"
+	"github.com/cockroachdb/cockroach/pkg/util/span"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/errors"
@@ -1356,7 +1356,11 @@ func TestAlterChangefeedAddTargetsDuringSchemaChangeError(t *testing.T) {
 func TestAlterChangefeedAddTargetsDuringBackfill(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.WithIssue(t, 141463)
+
+	// TODO(#141405): Remove this once llrbFrontier starts merging
+	// adjacent spans. The current lack of merging causes issues
+	// with the checks in this test around expected resolved spans.
+	defer span.EnableBtreeFrontier(true)()
 
 	var rndMu struct {
 		syncutil.Mutex

